README.md

```diff
@@ -1,38 +1,126 @@
 # lndmon
 
-#### A drop-in monitoring solution for your lnd node using Prometheus and Grafana.
+#### A drop-in monitoring solution for your lnd node that integrates with Prometheus and Grafana.
 
 <img src="dashboard.png">
 
 ## What is this?
 
-`lndmon` is a drop-in, dockerized monitoring/metric collection solution for your
-individual lnd nodes connected to bitcoin. With this system, you'll be able to
-closely monitor the health and status of your `lnd` node.
+`lndmon` is a drop-in, monitoring/metric collection solution for your
+individual lnd nodes connected to bitcoin which allows you to closely monitor
+the health and status of your `lnd` node.
 
-There are three primary components of the `lndmon` system:
+There are four primary components of a complete monitoring system:
```

Comment:
Nit: can replace with "There are four primary components of this monitoring system:"